samples/zephyr-hello/build.bat

```diff
@@ -0,0 +1,34 @@
+@echo off
+setlocal
+
+set BOARD_ARCH=arm_cortex_m3
+set BOARD_NAME=qemu_cortex_m3
+set TOOLCHAIN=gcc-arm-none-eabi-7-2017-q4-major-win32
+if not defined ZEPHYR_BASE (set ZEPHYR_BASE=%userprofile%\zephyr)
+
+set DIR=%~dp0
+if "%KONAN_DATA_DIR%"=="" (set KONAN_DATA_DIR=%userprofile%\.konan)
```

Comment:
Works on Windows too.
Nitpick (to me :)) - a little bit inconsistent, better
`if not defined  KONAN_DATA_DIR (set KONAN_DATA_DIR=%userprofile%\.konan)`